### PR TITLE
Streamer service

### PR DESCRIPTION
**Pull request by supra08**
@alphadose

### CODE REVIEW

File: services/jikan/controller.go [hunk 1]
```diff
@@ -0,0 +1,89 @@
...
+	"github.com/sdslabs/gasper/configs"
+	"github.com/sdslabs/gasper/lib/mongo"
+	"github.com/sdslabs/gasper/lib/utils"
+	"github.com/sdslabs/gasper/types"
+)
+
+// ServiceName is the name of the current microservice
+const ServiceName = types.Jikan
+
+func streamHandler(c *gin.Context) {
```
Comment (alphadose) [inline services/jikan/controller.go:23]: rename this to `streamMetrics` maybe

File: services/jikan/controller.go [hunk 2]
```diff
@@ -0,0 +1,89 @@
...
+	if err != nil {
+		metricsInterval = 2 * int64(configs.ServiceConfig.Mizu.MetricsInterval)
+	}
+
+	metricsCount, err := strconv.ParseInt(c.Query("count"), 10, 64)
+	if err != nil {
+		metricsCount = 10
+	}
+
+	chanStream := make(chan []types.M, 10)
```
Comment (alphadose) [inline services/jikan/controller.go:36]: Having a large write timeout, I think the channel is no longer necessary

Lets keep it simple and roll back to the original implementation. Spawning an extra goroutine and channel for every request is costly

**Comment by alphadose** [comment]
Also have a look as to why the CI is failing

**Comment by supra08** [comment]
Yeah this PR depends on a particular function in the `implement-SSE` branch. Once it is rebased with that, the CIs will pass.

**Pull request merged**